CHANGES.md

```diff
@@ -10,6 +10,12 @@ To be released.
 
  -  The `uri` type has completly gone; use `url` instead.
     [[#126], [#281] by Jonghun Park]
+ -  Support [external_tag][etag] via '@external-tag' annotation.
```

Comment:
Changelogs should be the past tense.  Also the following explanation does not have enough information:

> external_tag via '@external-tag' annotation